math/number_of_positive_divisors.cpp

```diff
@@ -12,11 +12,11 @@
  *
  * Example:- N=36
  * 36 = (3^2 * 2^2)
- * number-of-divisors(36) = (2+1) * (2+1) = 9.
- * list of divisors of 36 = 1, 2, 3, 4, 6, 9, 12, 18, 36.
+ * number_of_positive_divisors(36) = (2+1) * (2+1) = 9.
```

Comment:
Please also document the results of `number_of_positive_divisors(-36)` so that users know what to expect if they provide a negative number to your algorithm.

Reply:
Done !